src/types/streams/streams_bucket.rs

```diff
@@ -0,0 +1,28 @@
+use crate::types::profile::UID;
+use crate::types::streams::StreamsItem;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use std::collections::HashMap;
+
+#[derive(Default, Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize)]
+pub struct StreamsItemKey {
+    pub meta_id: String,
+    pub video_id: String,
+}
+
+#[serde_as]
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct StreamsBucket {
+    pub uid: UID,
+    #[serde_as(as = "Vec<(_, _)>")]
```

Comment:
What will be the json representation in this case?

I guess it will be in this form:
```json
[
    [
       { "meta_id": "...", "video_id": "..."},
       { ... StreamsItem ... }
    ]
]
```

So we need the fields in `camelCase` and wanted to check if this is the correct representation that we want for the `StreamsBucket.items`